core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -469,6 +605,40 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
       </button>
     </div>
   </script>
+
+  <script type="text/ng-template" id="modals/editorViewSuggestion">
```

Comment:
A note for the future: for comments like this (that contain questions), please answer the questions, or explain what you did. This makes the review go faster. Thanks!